optimum/onnxruntime/optimization.py

```diff
@@ -118,7 +122,11 @@ def optimize(
         self.config.save_pretrained(save_dir)
 
         # Create and save the configuration summarizing all the parameters related to optimization
-        ort_config = ORTConfig(optimization=optimization_config)
+        ort_config = ORTConfig(
+            optimization=optimization_config,
+            use_external_data_format=use_external_data_format,
+            all_tensors_to_one_file=all_tensors_to_one_file,
```

Comment:
Hey Ella, I added those two flags while saving the config, does that make sense to you?

Reply:
What is the advantage of setting `all_tensors_to_one_file` to `False` ? Also the name used by onnnxruntime is not super intuitive so I would be for renaming it to something clearer. It's good point to add the `use_external_data_format` to the ort config !

Reply:
Could this parameter be inferred (depending on the model size) instead of being explicitly set ?

Reply:
Thanks for the explanation, do you think you can add it in the docstrings ? Also the information that `all_tensors_to_one_file` is only relevant when `use_external_data_format` is set to `True`. And do you think we can rename `all_tensors_to_one_file` to something which explicitly link it to the external data format such as `one_external_file` or `not external_file_for_each_tensor` ?

Reply:
@echarlaix `one_external_file` sounds good to me, will update the name and add description of its relationship with `use_external_data_format` in the docstring.